diary-alexander-jaffray/modernized/15-notes-diary.adoc

```diff
@@ -2070,16 +2124,19 @@ Towards the beginning of the month called June, 1661,
 being accused in Parliament by the King`'s advocate of seditious practices,
 he was banished the King`'s dominions,
 without being heard by the Parliament in his own defence.
-He died in Holland.--Woodrow`'s
-History of the Sufferings of the Church of Scotland, vol. i. p. 71.
+He died in Holland.--Woodrow`'s [.book-title]#History of the Sufferings of the Church of Scotland,# vol. i. p. 71.
 
 === Note MM.
 
-In the Acts and Minutes of the Parliament of Scotland,
+In the _Acts and Minutes of the Parliament of Scotland,_
 under date of "`the 16th of January,
 1661,`" is recorded the following Petition of Alexander Jaffray,
 together with the decision of the house upon it.
 
+[.embedded-content-document.legal]
+--
+
+[.letter-heading]
 To the Lord Commissioner his Grace, and the honourable Estates of Parliament.
 
 The humble petition of Alexander Jaffray shows,
```

Comment:
Lets put an [.offset] here above: The humble petition of Alexander Jaffray shows,